source/browseMode.py

```diff
@@ -1284,8 +1284,6 @@ def script_shiftTab(self, gesture):
 	def event_focusEntered(self,obj,nextHandler):
 		if obj==self.rootNVDAObject:
 			self._enteringFromOutside = True
-		if self.passThrough:
-			 nextHandler()

```

Comment:
I think it'd be good to have a comment here explaining that we ignore these events here, but event_gainFocus will replay them if appropriate.